src/Chapter13/Listing13.12.CreatingDelegateWithStatementLambda.cs

```diff
@@ -1,8 +1,8 @@
-﻿// TODO: Update listing in Manuscript
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_12
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_12
 {
     using System;
-
+    #region INCLUDE
+    #region EXCLUDE
```

Comment:
Rather an include with two excludes to strip of the beginning and end, wouldn't it be simpler to just move the #region INCLUDE to just surround the inner section?